erpnext/stock/doctype/stock_settings/stock_settings.json

```
@@ -290,14 +291,20 @@
    "fieldname": "mr_qty_allowance",
    "fieldtype": "Float",
    "label": "Over Transfer Allowance"
+  },
+  {
+   "default": "0",
```

```suggestion
   "default": "0",
   "depends_on": "auto_insert_price_list_rate_if_missing",
```

This field should only appear if auto-insert is enabled 😅 

Else LGTM.